docs/modules/ROOT/pages/tmail-backend/configure/index.adoc

```diff
@@ -36,3 +36,15 @@ Documentation to generate a private key file for service account: https://fireba
 
 Example: `privatekey.url=/root/conf/firebase_push_privatekey.json`.
 |===
+
+== Additional Queue configurations
+Specified to TMail backend, we can configure the following configurations in the *queue.properties* file.
+
+.queue.properties additional content
+|===
+| Property name | explanation
+| event.bus.keys.choice
+| Optional. Default to the REDIS implementation. Supported values (case insensitive): `REDIS` / `RABBITMQ`.
```

Comment:
Do we want the Event bus keys implementation to default to Redis or RabbitMQ?

Reply:
I would say Rabbitmq.

Mainly for retro-compatibility with existing deployments, to not potentially have a mess on existing deployments when rolling this?

Reply:
switched default to rabbitmq